content/en/cloud_cost_management/tags/_index.md

```diff
@@ -0,0 +1,126 @@
+
+| Source | What tags are collected | Description |
+|---|---|---|
+| All Providers | Bill columns | Such as AWS Cost and Usage Report (CUR) columns, Google Billing Export columns, etc |
+| Datadog Enrichment | Host Agent | Tags added to host metadata by the Datadog agent running on the host |
+| Datadog Enrichment | Service Catalog | Tags associated with this service in the APM Service Catalog |
+| Datadog Enrichment | Integration Tiles | Tags added to the Datadog integration tile for a specific cloud account. Integration tile tags apply to all costs in that account. Requires enabling the provider integration for each account |
+| Datadog Enrichment | Data Observability | Tags from the Datadog’s Data Observability product, requires enabling BigQuery monitoring |
+| Datadog Enrichment | Cloud Network Monitoring | Source and destination dimensions from the Datadog Cloud Network product. Requires enabling Cloud Network in the Datadog agent |
+| Kubernetes Enrichment | Kubernetes Node | User-defined tags found on Kubernetes nodes monitored with Datadog | 
+| Kubernetes Enrichment | Kubernetes Pod | User-defined tags found on Kubernetes pods monitored with Datadog | 
+| Kubernetes Enrichment | Kubernetes Persistent Volume | User-defined tags found on Persistent Volumes in Kubernetes clusters monitored with Datadog |
+| Kubernetes Enrichment | Kubernetes Persistent Volume Claim | User-defined tags found on Persistent Volume Claims in Kubernetes clusters monitored with Datadog | 
+| Cloud Cost Management | Cloud Cost Aliases | Some aliases are tags derived from provider cost data, used to simplify the cost data model, such as `aws_product`, which is an alias of `lineItem/ProductCode`. Additional tags that exist on both cost data and integration metrics are added, allowing cost data and usage data to be combined in Custom Allocation Rules, dashboards, and notebooks |
+| Cloud Cost Management | Cloud Cost Allocation | Tags created during cost allocation that specify the split of shared resources, such as `allocated_spend_type` |
+| Cloud Cost Management | FOCUS | Provider-agnostic tags compliant with [FOCUS][8], the unifying format for cloud billing data |
+| Tag Pipelines | Rules defined by user | Tags created by applying your Tag Pipelines to cost data |
+| Custom Allocation Rules | Rules defined by user | Tags created by applying your Custom Allocation Rules to cost data (do not apply to SaaS costs) |
+
+Datadog also adds provider specific tags:
+
+| Provider | What tags are collected | Description |
+|---|---|---|
+| AWS | Cost Allocation Tags | Any tag defined by the user in [AWS Cost Allocation][1] tags that show up in the AWS CUR |
+| AWS | AWS Resource Groups Tagging API | User-defined tags on a cloud resource in AWS, pulled by Cloud Cost Management using the Group Tagging API |
```

Comment:
```suggestion
| AWS | AWS Resource Groups Tagging API | User-defined tags on a cloud resource in AWS, pulled by Cloud Cost Management using the Groups Tagging API |
```

and add this link - https://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/overview.html